Libs/GObject/Classes/Object.cs

```diff
@@ -25,14 +26,14 @@ public partial class Object : INotifyPropertyChanged, IDisposable
         #endregion
 
         #region Properties
-
-        protected internal IntPtr Handle { get; private set; }
+        
+        public IntPtr Handle { get; private set; }
```

Comment:
The Handle property became public (again 😅)

Reply:
This one is sort-of intentional (See: https://github.com/gircore/gir.core/pull/105/commits/9c9e45a3852eee3dba77d166bce1a4b743cd9e6d)

The commit message:

```
It turns out that we cannot use `protected internal` after all when accessing a less derived class from a more-derived
class (e.g. Accessing Widget.Handle in Box).

See CS1540: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1540

Also makes Disposed private, as it is not used outside the class.
```

I wanted to clarify this, because I could just be missing something. Could you try using `protected internal` again and see if it builds? The problem I'm experiencing is where we can't access Handle from a more-derived version of a type (e.g. we can go Widget->Box, but not Box->Widget).

Reply:
Isn't for that cases the `GetHandle` method is used ? To do `GetHandle(widget)` in the `Box` class for example.

Reply:
Hmm. Seems odd that `GetHandle()` would work, but not `Handle` directly.

Maybe we should consider making `Handle` private and forcing `GetHandle()` for everything to be consistent? (For another PR, of course).

Anyways, I'll push another commit that changes this.

Reply:
> Maybe we should consider making Handle private and forcing GetHandle() for everything to be consistent? (For another PR, of course).

I also think is the better solution, I'm already using `GetHandle` for everything in #103

Reply:
I edited that comment thinking we'd probably want to access a type's own Handle (hence the protected), but you raise a good point. There's no reason why we can't use the accessor method, so it might be best to enforce this.